scripts/capstone.py

```diff
@@ -182,7 +182,7 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
         args = kwargs["arguments"]
         show_opcodes = args.show_opcodes
         length = args.length or gef.config["context.nb_lines_code"]
-        location = parse_address(args.location)
+        location = int(gdb.parse_and_eval(args.location).address)
```

Comment:
```suggestion
        location = location = int(parse_address(args.location))
```

**int(gdb.parse_and_eval(args.location).address)** is not working